core/domain/feedback_jobs_one_off.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""One-off jobs for feedback models."""
+
+from core import jobs
+from core.platform import models
+
+(feedback_models,) = models.Registry.import_models([models.NAMES.feedback])
+
+
+class GeneralFeedbackThreadUserOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Don't forget to add the job to job_registry so that you can run it from the admin dashboard.